src/scss/components/_toast.scss

```diff
@@ -4,8 +4,8 @@
   align-items: flex-start;
   position: fixed;
   width: 30%;
-  bottom: 20px;
-  z-index: 1000;
+  bottom: 45px;
+  z-index: 999999;
```

Comment:
Isn't be better to use 1001 instead?

Reply:
No because of the z-index inside the blip-project